lib/propolis/src/hw/virtio/viona.rs

```diff
@@ -648,6 +687,14 @@ impl VionaHdl {
         Ok(())
     }
 
+    fn set_notify_iop(&self, port: Option<u16>) -> io::Result<()> {
```

Comment:
Perhaps this is just because I don't have an illumos background, but it took a bit for me to realize that "IOP" here referred to "IO port"...I might prefer to expand it to `set_notify_io_port`? I note that we have a function above `notify_port_update` that calls this function, so perhaps we could call this `set_notify_port` to mirror that? But, if you prefer to more closely mirror the name of the ioctl, that's fair, too. :)